src/users.cpp

```diff
@@ -1248,9 +1248,7 @@ const std::string& FakeUser::GetFullHost()
 
 const std::string& FakeUser::GetFullRealHost()
 {
-	if (!ServerInstance->Config->HideServer.empty())
-		return ServerInstance->Config->HideServer;
-	return server->GetName();
+	GetFullHost();
```

Comment:
This should be returned.

Reply:
@SadieCat, fix it.
Commit info here:
https://github.com/inspircd/inspircd/pull/1879/commits/b4e0c5b6212def02386af0a555445412eed6f1f4